pymedphys/_data/urls.json

```diff
@@ -30,5 +30,6 @@
   "rtplan-anonymisation.zip": "https://zenodo.org/record/3930533/files/rtplan-anonymisation.zip?download=1",
   "treatment-record-anonymisation.zip": "https://zenodo.org/record/3931186/files/treatmentrecord-anonymisation.zip?download=1",
   "negative-mu-density.trf": "https://zenodo.org/record/3973722/files/negative_mu_density.trf?download=1",
-  "trf-references-and-baselines.zip": "https://zenodo.org/record/3987725/files/trf-references-and-baselines.zip?download=1"
+  "trf-references-and-baselines.zip": "https://zenodo.org/record/3987725/files/trf-references-and-baselines.zip?download=1",
+  "dicom-to-delivery-issue-#1047.zip": "https://github.com/pymedphys/pymedphys/files/5212087/NPCA_Anonymised.zip"
```

Comment:
why not zenodo?

Reply:
The data was uploaded by a user over at https://github.com/pymedphys/pymedphys/issues/1047#issuecomment-691389682

...didn't feel right to re-upload to a different location given it wasn't my data... In the past I have asked users to upload to Zenodo themselves, but the barrier to entry is larger than "drag and drop onto a GitHub issue".

Reply:
Fair enough.